src/components/mapBox/MapBox.tsx

```diff
@@ -1,17 +1,20 @@
 import { IconLinkExternal, IconLocation } from "hds-react";
 import React from "react";
 import { useTranslation } from "next-i18next";
+import { IconAngleDown } from "hds-react";
 
 import Text from "../text/Text";
 import InfoBlock from "../infoBlock/InfoBlock";
 import styles from "./mapBox.module.scss";
+import { AccessibilitySentences } from "../../types";
 
 type Props = {
   title: string;
   serviceMapUrl: string;
   placeName: string;
   placeAddress: string;
   links?: React.ReactElement<React.ComponentProps<typeof InfoBlock.Link>>[];
+  accessibilitySentences?: AccessibilitySentences[];
```

Comment:
I think that passing the accessibility data as is does not necessarily make sense here as you've gone through the effort of implementing the `Collapse` content model for `InfoBlock`. Instead of passing `accessibilitySententences`, perhaps it should be an `InfoBlock.Collapse` element that gets passed.

I'd also consider whether, depending on what your intention was, we should only have a single prop of type `InfoBlockContent[]`, which then allows us to pass the links and the collapsed content.